cmake/recipes/libigl.cmake

```diff
@@ -20,8 +20,11 @@ message(STATUS "Third-party: creating target 'igl::core'")
 include(FetchContent)
 FetchContent_Declare(
     libigl
-    GIT_REPOSITORY https://github.com/libigl/libigl.git
-    GIT_TAG 3ea7f9480967fcf6bf02ce9b993c0ea6d2fc45f6
+    #GIT_REPOSITORY https://github.com/libigl/libigl.git
+    #GIT_TAG 3ea7f9480967fcf6bf02ce9b993c0ea6d2fc45f6
+    GIT_REPOSITORY https://github.com/mtao/libigl.git
```

Comment:
This change needs to be undone before the final submission - this branch is a working set of updates to libigl to suppress some unnecessary warnings on gcc 11.3